test/e2e/specs/login.js

```diff
@@ -0,0 +1,71 @@
+const LoginPage = require('../page-objects/login');
+
+describe('Login Page', () => {
+	beforeEach(() => {
+		LoginPage.open();
+	});
+
+	it('should show the header, login box , and all login box elements', () => {
+		expect(LoginPage.header.isVisible()).to.equal(true);
+		expect(LoginPage.loginBox.isVisible()).to.equal(true);
+		expect(LoginPage.email.isVisible()).to.equal(true);
+		expect(LoginPage.password.isVisible()).to.equal(true);
+		expect(LoginPage.login.isVisible()).to.equal(true);
+		expect(LoginPage.forgotPassword.isVisible()).to.equal(true);
+		expect(LoginPage.forgotPassword.getAttribute('href')).to.equal('/reset');
+		expect(LoginPage.register.isVisible()).to.equal(true);
+		expect(LoginPage.register.getAttribute('href')).to.equal('/register');		
+	});
+
+	it('should show an error when no email is provided on a blur event', () => {
+		LoginPage.email.hasFocus() = true;
```

Comment:
`hasFocus` will return a boolean, so won't be assignable. There's no `setFocus` or `focus` method in webdriver.io, so you should just use `click`, then to blur away from it just click the password field or something.

Come to think of it, I don't think I sent on the docs for webdriver.io. Here's the docs for the click function: http://webdriver.io/api/action/click.html

Reply:
This needs to be done for all the below tests where you do the same thing.

Reply:
makes sense, thanks